ziti-tunnel/docker/docker-entrypoint.sh

```diff
@@ -49,8 +49,21 @@ if [ ! -f "${json}" ]; then
     ziti-tunnel enroll --jwt "${jwt}" --out "${json}"
 fi
 
-echo "running ziti-tunnel"
-set -x
-ziti-tunnel -i "${json}" "${@}" &
-ZITI_TUNNEL_PID=$!
-wait $ZITI_TUNNEL_PID
+# use iptables-legacy if nftables fails to accept iptables translations
+iptables-nft -t mangle -S --wait &>/dev/null || {
```

Comment:
From [the Slack scrollback](https://netfoundry.slack.com/archives/C525F9Q3Y/p1605039773480400?thread_ts=1604952299.440200&cid=C525F9Q3Y):
> The [source] container image (`debian:slim-stable`) has an update-alternatives precedence resolving `iptables` => `/usr/sbin/xtables-nft-multi` which is the `iptables` compatibility interface to `nft` and which encounters an error when attempting to exec the `nft` translations that were passed through `iptables` because `nft` isn't available on the host that is running Docker Engine.

And so, the rationale at the time was to specifically probe for the availability of `nft` on the host by attempting to exec `iptables-nft` in the container. In light of your observation that there are many ways for running `iptables-nft` to fail, I agree we need a better way of probing for `iptables` compatibility. I've reconciled the source branch here with your suggestions. See if that gives you warm fuzzies or any better ideas. 

Now I'm trying to think of a way to test this is working on ARMv7 and v8 since I don't have that hardware at my fingertips. I believe I can exec a particular manifest with qemu. :thinking: